docs/source/getting_started.rst

```diff
@@ -1,2 +1,155 @@
+  pip install -r requirements.txt
+
+
+***********************
+Create a Django Project
+***********************
+
+Orchestra is a Django app, which means that it must be run within a Django
+project (read up at `the tutorial
+<https://docs.djangoproject.com/en/1.8/intro/tutorial01/#creating-a-project>`_
+if that's confusing). Start a project with
+``django-admin startproject your_project``, replacing ``your_project`` with
+your favorite project name. From here on out, this document will assume that
+you stuck with ``your_project``, and you should replace it appropriately.
+
+
+*******************************
+Install and Configure Orchestra
+*******************************
+
+Next, let's get Orchestra installed and running. To get the code, just install
+using pip: ``pip install orchestra``.
+
+Orchestra has a number of custom settings that require configuration before
+use. First, download the default orchestra settings file and place it next to
+the project settings file::
+
+  wget https://raw.githubusercontent.com/unlimitedlabs/orchestra/v0.1.0/example_project/example_project/orchestra_settings.py
+  mv orchestra_settings.py your_project/your_project
+
+Next, edit the ``orchestra_settings.py`` file:
+
+* Add ``'simple_workflow'`` to ``settings.INSTALLED_APPS`` in the "General"
+  section if you want to run the demo workflow
+  (:ref:`instructions below <demo-section>`).
+
+* Change settings like the ``ORCHESTRA_PROJECT_API_SECRET`` from ``'CHANGEME'``
+  to more appropriate values.
+
+* Add 3rd party credentials in the "3rd Party Integrations" section so that
+  Orchestra can store files on `Amazon s3 <https://aws.amazon.com/s3/>`_ and use
+  `Google apps <http://apps.google.com>`_ and `Slack <https://slack.com/>`_ to
+  help communicate with expert workers (though these are all optional).
+
+Then, at the bottom of your existing settings file
+(``your_project/your_project/settings.py``), import the orchestra
+settings::
+
+  from .orchestra_settings import setup_orchestra
+  setup_orchestra(__name__)
+
+You'll also need to set up orchestra's URLs, so that Django knows where to
+route users when they view orchestra in the browser. Add the following code
+inside the ``urlpatterns`` variable in
+``your_project/your_project/urls.py``::
+
+    # Registration Views
+    # Eventually these will be auto-registered with the Orchestra URLs, but for
+    # now we need to add them separately.
+    url(r'^orchestra/accounts/',
+        include('registration.backends.default.urls')),
+
+    # Logout then login is not available as a standard django
+    # registration route.
+    url(r'^orchestra/accounts/logout_then_login/$',
+        auth_views.logout_then_login,
+        name='logout_then_login'),
+
+    # Orchestra URLs
+    url(r'^orchestra/',
+        include('orchestra.urls', namespace='orchestra')),
+
+    # Beanstalk Dispatch URLs
+    url(r'^beanstalk_dispatch/',
+        include('beanstalk_dispatch.urls')),
```

Comment:
Added the admin.

The bare `urls.py` isn't importable, since it lives in `example_project`, not `orchestra`. It's a TODO to refactor the urls so we don't need this many changes to the top-level `urls.py`. The `wget` approach I just added might simplify it somewhat?

If you think urls should be refactored before the release, let me know.